allel/io/vcf_read.py

```diff
@@ -238,6 +238,10 @@ def read_vcf(input,
              log=None):
     """Read data from a VCF file into NumPy arrays.
 
+    .. versionchanged:: 1.12.0
+        Now returns None if no variants are found in the VCF file or matching the
+        requested region.
+
     Parameters
     ----------
     input : string or file-like
```

Comment:
PEP8 suggests not using existing function names as variables.

Reply:
Yeah, I should have done something different, but I think we're stuck with this for now to maintain backwards compatibility with anyone who has code using "input" as a keyword argument.